ehrql/tables/tpp.py

```diff
@@ -552,6 +553,160 @@ class clinical_events_ranges(EventFrame):
     )
 
 
+@table
+class covid_therapeutics(EventFrame):
+    """
+    The COVID Therapeutics dataset contains information on COVID treatments used in inpatient
+    and outpatient settings.
+
+    **Metadata**
+
+    * **Data provider** NHS England
+    * **Participation / Coverage** Inpatients and outpatients treated with antivirals/nMABs for COVID-19 in England
+    * **Provenance** Data sourced largely from BlueTeq system (forms completed by clinicians)
+    * **Update frequency in OpenSAFELY** Approximately weekly
+    * **Delay between event occurring and event appearing in OpenSAFELY** Approximately 2-9 days
+    * **Collected information** Treatment start date; therapeutic intervention; COVID indication, current status, risk group, region
+
+
+    **Overview**
+
+    Antivirals and neutralising monoclonal antibodies (nMABs) for COVID-19 can be
+    administered in inpatient setting or, for outpatients, in COVID Medicine Delivery
+    Units (CMDUs) specifically set up for this purpose. For patients considered for
+    these treatments, clinicians submit completed forms to NHS England. Each row
+    represents one completed form for one course of treatment. Data received by
+    OpenSAFELY currently covers patients who were approved for treatment. The patient
+    may or may not have actually received the treatment or completed the course (but we
+    assume that they usually do). They may have another form completed for another
+    treatment, either because it was decided to give them a different treatment, or for
+    some other reason. They may in theory also have another form completed some months
+    later for another instance of infection.
+
+    Treatment dates may be in the past or future at the point when the form is
+    submitted.
+
+    Note that this dataset may contain **duplicate** rows – full duplicates are removed
+    but there may remain some partial duplicates.
```

Comment:
We need to document that the risk group only applies to Sotroviman, Molnupiravir, and Casirivimab & imdevimab interventions (see comments on issue #2023)
It's probably a good idea to get Millie to review the docs for this table?

Reply:
I mentioned this on the [series description](https://github.com/opensafely-core/ehrql/pull/2025/files#diff-a3e087910f839341b26b122527c775d2bf29f55802a35228b3307436da676065R651-R652) although I'm going to revise that wording to mention intervention names rather than db column names. But do you think it should be in the table description here as well, or is just in the series description sufficient?

Reply:
I think the series description is probably fine :+1: